src/components/ReduxFormCheckbox/ReduxFormCheckbox.test.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import ReduxFormCheckbox from './index';
+
+describe('<ReduxFormCheckbox />', () => {
```

Comment:
Also run this by Lael but I question the value of this snapshot test that only checks that checking/unchecking changes the reduxformcheckbox markup. Isn't that just part of the component's features do we need to test it?

I think perhaps it's more usefl to test that checking/unchecking it changes the state of the params that are sent to the server? Did we already test that separately?

Reply:
There may be additional context or nuance I'm missing, but from what I experienced trying to test Bulk Enrollment, Redux doesn't store things as accessible state, it passes them around as props. Thus, you can't really do an integration test of the "pretend I'm a user and click things" variety. Everything would be passed around as props, so a snapshot test is a clean version of verifying the checkbox renders when passing in relevant props.

Reply:
Agree.
I'm gonna work on better testing of these forms when I refactor. In their current state, integration testing is well nigh impossible.

Reply:
Basically, this tests: if we pass the right props to this component, does it do the right thing? 
We're input.checked here, that's what needs to be tested, everything else is a responsibility either of Paragon or Redux-Forms